src/main/controller/SvgFile.java

```diff
@@ -0,0 +1,18 @@
+package controller;
+
+import java.util.List;
+
+import model.Shapes;
+
+public interface SvgFile {	
+	
+	//if filename does not exist, create a new svg file
+	//if filename exists, then write content into it
+	public boolean writeSVGFile(String filename, String content);
+	
+	//add shapes to a svg file
+	public boolean addShapes(List<Shapes> listShapes);
```

Comment:
For me, content is the String we will write into a svg file. Returning a boolean will indicate if the String was successfully written into the ile or not. 
Can you simply write the signature of the method that you suggest? It would be clearer.